service/resource/legacy/resource.go

```diff
@@ -112,60 +102,6 @@ func (r *Resource) GetCreateState(ctx context.Context, obj, currentState, desire
 		spec = o.Spec
 	}
 
-	// Create flannel etcd config.
```

Comment:
Here we can now remove the etcd magic because it is done in the separate resource.